docs/includes/eloquent-models/SchemaVersionTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests;
+
+use App\Models\Planet;
+use MongoDB\Laravel\Tests\TestCase;
+
+class SchemaVersionTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testSchemaVersion(): void
+    {
+        require_once __DIR__ . '/PlanetSchemaVersion.php';
+
+        Planet::truncate();
+
+        $saturn = Planet::create([
+            'name' => 'Saturn',
+            'type' => 'gas',
+        ]);
+
+        $wasp = Planet::create([
+            'name' => 'WASP-39 b',
+            'type' => 'gas',
+            'schema_version' => 1,
+        ]);
+
+        $planets = Planet::where('type', 'gas')
+            ->get();
+
+        echo 'After migration:\n' . $planets;
```

Comment:
`echo` should be avoided in tests. Otherwise you need to assert the output: https://github.com/mongodb/laravel-mongodb/blob/a62d4b91796b00c2cf46845bab9fd23cc0df6a97/docs/includes/usage-examples/CountTest.php#L36-L41